run_linux.go

```diff
@@ -704,12 +704,14 @@ func runUsingRuntime(isolation define.Isolation, options RunOptions, configureNe
 	// Lock the caller to a single OS-level thread.
 	runtime.LockOSThread()
 
+	logger := options.Logger
```

Comment:
This never needs to be different from `options.Logger`, so drop the variable and use the passed-in value.